README.md

```diff
@@ -158,6 +158,7 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
 - [plash](https://github.com/ihucos/plash) - A container run and build engine - runs inside docker.
 - [rocker-compose](https://github.com/grammarly/rocker-compose) - Docker composition tool with idempotency features for deploying apps composed of multiple containers. By [@grammarly](grammarly)
 - [rocker](https://github.com/grammarly/rocker) - Extended Dockerfile builder. Supports multiple FROMs, MOUNTS, templates, etc. by [grammarly](grammarly).
+- [Smith](https://github.com/oracle/smith) - A Micocontainer Builder and can perform multi-stage builds after the image is built [Oracle](https://github.com/oracle)
```

Comment:
Thanks! But wouldn't it be better in `image -> builder` ?

Reply:
Also, the link `[Oracle](https://github.com/oracle)` will be duplicated because the PR #600 got merged. Can it be added as a link reference ? 
like `[Oracle][oracle]`
and in the end of the file:
`[oracle]: https://github.com/oracle`